subsys/bluetooth/host/Kconfig

```diff
@@ -351,6 +351,24 @@ config BT_OOB_DATA_FIXED
 	  value. This option should only be enabled for debugging and should
 	  never be used in production.
 
+config BT_KEY_DISPLACEMENT
```

Comment:
I'd call this something else, perhaps a bit more descriptive of the policy it implements. Perhaps `BT_KEYS_OVERWRITE_OLDEST`?